packages/swapper/package.json

```diff
@@ -21,6 +21,7 @@
     "swapcli": "yarn build && node ./dist/swappercli.js"
   },
   "dependencies": {
+    "@shapeshiftoss/asset-service": "^1.3.0",
```

Comment:
Is this on purpose? It doesn't look like your make any other changes in `swapper`.

Reply:
yeah - it doesn't build otherwise. i could move this to a devDependency as it's only used by swapper cli